### ISSUE TITLE
Build a Checkout Page Lab has redundant aria-required step

### ISSUE THREAD

**Issue by kaipepler** [freeCodeCamp/freeCodeCamp#59829]
### Describe the Issue

Step 9 of the "Build a Checkout Page" lab tells users "At least two of your input elements should be required and have an `aria-required` attribute set to `true`."

But `aria-required` is unnecessary here since they're already using the base html `required` attribute. This is reflected in the source code of the example project as well.

### Affected Page

https://www.freecodecamp.org/learn/full-stack-developer/lab-checkout-page/build-a-checkout-page

### Your code

```

```

### Expected behavior

"At least two of your input elements should be required."

I believe this would be better if that last step were simplified, removing the `aria-required` part so students new to accessible design don't think that they need to go through their forms and add both to all their required inputs.

### System

- Device: Laptop
- OS: macOS 15.4.1
- Browser: Safari
- Version: 18.4

**Comment by kaipepler** [comment]
Apologies, I just noticed this is already covered in #59725, I'll close my thread.
